Description: Avoid implicit function declarations

For compatibility with future compilers which do not support them
by default.

Include <ctype.h> for toupper, "moveresize.h" for
moveresize_setup_tab, and add a desktops_setup_tab prototype to
"desktops.h" because it is called from main.c.

Author: Florian Weimer
Bug-Debian: https://bugs.debian.org/1066381
Origin: https://bugzilla.icculus.org/show_bug.cgi?id=6671
Forwarded: yes

--- a/src/appearance.c
+++ b/src/appearance.c
@@ -19,2 +19,3 @@
 
+#include <ctype.h>
 #include "main.h"
--- a/src/desktops.h
+++ b/src/desktops.h
@@ -26,2 +26,3 @@
 void desktops_setup_names(GtkWidget *w);
+void desktops_setup_tab(void);
 
--- a/src/main.c
+++ b/src/main.c
@@ -26,2 +26,3 @@
 #include "mouse.h"
+#include "moveresize.h"
 #include "desktops.h"
